specs/commandline/sign.md

```diff
@@ -82,6 +83,16 @@ notation sign --signature-format cose <registry>/<repository>@<digest>
 notation sign <registry>/<repository>@<digest>
 ```
 
+### Sign an OCI Artifact with user metadata
+
+```shell
+# Prerequisites: 
+# A default signing key is configured using CLI "notation key"
+
+# sign an artifact stored in a registry and add user-metadata io.wabbit-networks.buildId=123 to the payload
+notation sign <registry>/<repository>@<digest> --user-metadata io.wabbit-networks.buildId=123
```

Comment:
Can we add an example for adding multiple user metadata? I suppose it is something like 
`--user-metadata io.wabbit-networks.buildId=123 --user-metadata io.wabbit-networks.buildTime="xxx"`